.docs/user-guide/demo.md

```diff
@@ -0,0 +1,153 @@
+# Demo
+
+## REX-Ray with Two-Node libStorage Client/Server
+
+This demo consists of a two-node deployment with the first node configured as a REX-Ray/libStorage server and the second node as merely a client. Both nodes have Docker installed and configured to leverage REX-Ray for persistent storage.
```

Comment:
i think i fixed the 80-character count. please let me know if i did it correctly.

you get the idea ;)